nemo/export/quantize/quantizer.py

```diff
@@ -167,80 +98,73 @@ def dummy():
         set_data_parallel_group(mpu.get_data_parallel_group())
         set_tensor_parallel_group(mpu.get_tensor_model_parallel_group())
 
+        model.set_inference_config(OmegaConf.to_container(inference_config))
+
     @staticmethod
-    def _load_and_modify_config(
-        model_file: str,
-        tensor_model_parallel_size: Optional[int] = None,
-        pipeline_model_parallel_size: Optional[int] = None,
-    ):
-        model_cfg = load_config(model_file)
-
-        with open_dict(model_cfg):
-            model_cfg.activations_checkpoint_method = None
-            model_cfg.activations_checkpoint_granularity = None
-            model_cfg.sequence_parallel = False
-            if tensor_model_parallel_size is not None:
-                model_cfg.tensor_model_parallel_size = tensor_model_parallel_size
-            if pipeline_model_parallel_size is not None:
-                model_cfg.pipeline_model_parallel_size = pipeline_model_parallel_size
+    def modify_model_config(model_config: DictConfig) -> DictConfig:
+        with open_dict(model_config):
+            model_config.activations_checkpoint_method = None
+            model_config.activations_checkpoint_granularity = None
+            model_config.sequence_parallel = False
             # Only custom ModelOpt spec is supported for PTQ: this custom spec is largely based on local Megatron-LM
             # layer definitions to avoid Transformer Engine implementations that are currently not supported.
             # This layer spec also requires RoPE fusion to be disabled for tensor view operations in attention
             # layer implementation from megatron/core/transformer/dot_product_attention.py to be functional.
-            model_cfg.name = "modelopt"
-            model_cfg.apply_rope_fusion = False
+            model_config.name = "modelopt"
+            model_config.apply_rope_fusion = False
 
-        return model_cfg
+        return model_config
 
     @staticmethod
-    def _sample_output(model):
```

Comment:
This function is just moved down close to export where it is used for better readability